frontend-implicaction/src/app/forums/components/create-post-form/create-post-form.component.html

```diff
@@ -9,7 +9,8 @@
       </div>
     </div>
     <div class="col-md-6 d-flex align-items-center justify-content-end">
-      <p-dropdown [(ngModel)]="selectedGroup" [ngModelOptions]="{standalone: true}" [options]="groups"
+      <p-dropdown *ngIf="currentGroup === undefined" [(ngModel)]="selectedGroup" [ngModelOptions]="{standalone: true}"
```

Comment:
```suggestion
      <p-dropdown *ngIf="!currentGroup" [(ngModel)]="selectedGroup" [ngModelOptions]="{standalone: true}"
```
je pense que ça marche ça 👍